### PR TITLE
Feature/review comments emails

### PR DESCRIPTION
**Pull request by gruiz17**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/reviewninja/review.ninja/pull/942" target="_blank">ReviewNinja

File: src/server/templates/reviewThreadOpened.ejs [hunk 1]
```diff
@@ -0,0 +1 @@
+A review thread has been opened <strong>starred</strong> by <i><%= sender.login %></i> on <i><%= pullrequestname %></i> for <i><%= user %>/<%= repo %></i>.
\ No newline at end of file
```
Comment (dfarr) [inline src/server/templates/reviewThreadOpened.ejs]: Take a look at this !fix

Comment (dfarr) [inline src/server/templates/reviewThreadOpened.ejs]: Also, I'm not sure if we should call them "review threads", we never refer to them as such. Perhaps "line notes", or something along this line.

Comment (dfarr) [inline src/server/templates/reviewThreadOpened.ejs]: !fixed

**Comment by dfarr** [comment]
We should always log when flags are **created**.

We should send emails in two circumstances:
- a review thread has been **started** (aka currently closed or stateless review thread has been flagged !fix or !resolve)
- **all** review threads have been closed

**Comment by dfarr** [comment]
Let's create a migration that takes the value of `user:issue` and sets `user:review_thread` to this value.

**Pull request closed without merging**